Studiosity/avataaars
# DEV-7337 implement studiosity hat

#### What's this PR do?
Adds the studiosity hat as a topType asset and adds Tutor as an avatar style with the yellow circle
#### Why is it needed?
So that we can use the hat which is a custom asset and the circle
#### Where should the reviewer start?
Unfortunately it appears my transpiler is more recent and its updated ALL the dist files 😞. So go the files changed, search for package-lock and start from there, everything else can be ignored
#### How should this be manually tested?
- Follow the instructions in the readme on how to use it locally with eureka
- put the avatar create component somewhere in eureka (ive been using admin-document-verification.es6, just change the value to AvatarCreator and go admin >> specialists >> specialist >> documents)
- In eureka, modify the avatar-options.json and add StudiosityTutorHat as a topType (you will need to run `./bin/webpack` again)
- go the the avatar creator and verify the studiosity hat appears as a top option, verify it is selectable
- verify circle around the display avatar is blue
- got to `app/webpacker/src/react_components/document-verification/document-viewer.jsx` and change `avatarStyle` prop to `AvatarStyle.Tutor` on the Avatar component
- refresh the page, verify circle is now yellow
#### What are the relevant tickets?
[DEV-7337](https://yourtutor.atlassian.net/browse/DEV-7337)
